cmake/developer_package/packaging/changelog

```diff
@@ -0,0 +1,17 @@
+openvino (2022.2.0) unstable; urgency=low
+
+  [Ilya Lavrenov]
+  * Implemented Debian packages support
+  * Support of Ubuntu 22.04
+  * Support of python 3.10
+  * Introduced OpenVINO Runtime 2.0 C API
+
+ -- OpenVINO Developers <openvino@intel.com>  Thu, 28 Jul 2022 04:17:51 +0800
+
+openvino (2022.1.0) unstable; urgency=low
+
+  [Ilya Lavrenov]
```

Comment:
If I remember right before we had checks which parse code in order to avoid any names in code or comments...

Reply:
we parsed only commit authors to prevent usage of personal emails

Reply:
Will update in next PR

Reply:
Fixed here https://github.com/openvinotoolkit/openvino/pull/12416